packages/web/index.js

```diff
@@ -80,7 +80,11 @@ module.exports = (neutrino, opts = {}) => {
     };
   }
 
-  if (!options.targets.node && !options.targets.browsers) {
+  if (options.targets === false) {
+    options.targets = {};
+  } else if (options.targets.browsers === false) {
```

Comment:
Yeah - I did it this way because it seemed like `options.targets` had other props besides browsers that maybe you'd want to set, but still use the browerslistrc. Not totally sure of that's valid though, will have to check when I'm back on a machine: https://babeljs.io/docs/en/babel-preset-env#options